flow/aria.js

```diff
@@ -245,7 +258,8 @@ type ARIAProperty =
   | 'aria-valuetext'
   | ARIAState;
 
-type ARIAPropertyMap = {
+// {| [AriaProperty]?: mixed |}
+type ARIAPropertyMap = {|
```

Comment:
Making this exact ensures that we don't forget to add new aria properties to the types.

Reply:
love it